### PR TITLE
View context menu

### PR DESCRIPTION
**Pull request by mustafapc19**
UI:
image

### CODE REVIEW

**Comment by o1lab** [comment]
Ok - looks good.

Docs would need update. And also good to have the same menu in three dots of view.

File: packages/nc-gui/components/dlg/ViewCreate.vue [hunk 1]
```diff
@@ -308,7 +308,7 @@ onMounted(async () => {
           <NcSelect
             v-model:value="form.fk_grp_col_id"
             class="w-full nc-kanban-grouping-field-select"
-            :disabled="groupingFieldColumnId || isMetaLoading"
+            :disabled="isMetaLoading"
```
Comment (wingkwong) [inline packages/nc-gui/components/dlg/ViewCreate.vue:311]: double check if this change should apply to L328 as well

File: packages/nc-gui/components/smartsheet/toolbar/ExportSubActions.vue [hunk 2]
```diff
@@ -99,17 +100,30 @@ const exportFile = async (exportType: ExportTypes) => {
 </script>
 
 <template>
-  <a-menu-item>
-    <div v-e="['a:download:csv']" class="nc-base-menu-item" @click="exportFile(ExportTypes.CSV)">
-      <component :is="iconMap.csv" />
+  <div class="flex py-3 px-4 font-bold uppercase text-xs text-gray-500">Download Data</div>
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/toolbar/ExportSubActions.vue:103]: i18n

File: packages/nc-gui/components/smartsheet/toolbar/OpenedViewAction.vue [hunk 3]
```diff
@@ -0,0 +1,384 @@
+<script lang="ts" setup>
```
Comment (wingkwong) [inline packages/nc-gui/components/smartsheet/toolbar/OpenedViewAction.vue:1]: some functions are exactly same in `ViewActions.vue`. May centralise them.

Comment (mustafapc19) [reply packages/nc-gui/components/smartsheet/toolbar/OpenedViewAction.vue:1]: This UX is still in experimental phase. After some time will analyse and see if ppl are also using the top bread crumb view info dropdown that much, if not might be removed. So we can pick up centralising this logic at that time

**Comment by dstala** [comment]
- [ ] Download as XLSX => Change to `Download EXCEL` [to keep it uniform with upload menu]
- [ ] Download as CSV => `Download CSV` [remove as]
Screenshot 2023-11-09 at 7 06 06 PM
Screenshot 2023-11-09 at 7 05 55 PM


- [ ] Error when accessing context menu for grid view from left sidebar
```
Uncaught (in promise) Error: Please call `useProvideSmartsheetStore` on the appropriate parent component
    at useSmartsheetStoreOrThrow (useSmartsheetStore.ts:88:21)
    at setup (ExportSubActions.vue:36:1)
    at callWithErrorHandling (runtime-core.esm-bundler.js:158:18)
    at setupStatefulComponent (runtime-core.esm-bundler.js:7289:25)
    at setupComponent (runtime-core.esm-bundler.js:7250:36)
    at mountComponent (runtime-core.esm-bundler.js:5646:7)
    at processComponent (runtime-core.esm-bundler.js:5612:9)
    at patch (runtime-core.esm-bundler.js:5087:11)
    at ReactiveEffect.componentUpdateFn [as fn] (runtime-core.esm-bundler.js:5755:11)
    at ReactiveEffect.run (reactivity.esm-bundler.js:178:19)
```

**Comment by dstala** [comment]
- [ ] Error when switching from Locked > Colalborative : SQLite root DB
```
Uncaught (in promise) TypeError: Cannot set properties of null (setting 'Sortable1699608962571')
    at Sortable.destroy (sortable.esm.js:2458:15)
    at sort.ts:55:42
    at callWithErrorHandling (runtime-core.esm-bundler.js:158:32)
    at effect2.onStop (runtime-core.esm-bundler.js:1783:7)
    at ReactiveEffect.getter [as fn] (runtime-core.esm-bundler.js:1762:11)
    at ReactiveEffect.run (reactivity.esm-bundler.js:178:19)
    at Array.job (runtime-core.esm-bundler.js:1825:14)
    at flushPostFlushCbs (runtime-core.esm-bundler.js:330:40)
    at flushJobs (runtime-core.esm-bundler.js:368:5)
```

- [ ] Form view, exclude upload & download options
Screenshot 2023-11-10 at 3 08 45 PM

**Pull request merged**